packages/hoprd/src/api/v2/paths/sign.ts

```diff
@@ -0,0 +1,44 @@
+import { Operation } from 'express-openapi'
```

Comment:
Wouldn't this make more sense as a path like `/api/v2/message/sign` ?

Reply:
Yeah, wasn't sure whether we were already nesting routes, but happy to update this one.

Reply:
Solved in 094f1b2